recipes/trata/meta.yaml

```diff
@@ -0,0 +1,83 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 16 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "trata" %}
+{% set version = "1.0.2" %}
```

Comment:
It is tagged as 1.0.2 but the metadata in 1.0.1. See https://github.com/LLNL/trata/blob/95b9a404d0382f233c7e0a4e73f8b528b2bd1f3f/pyproject.toml#L4

Reply:
I'm not sure what is the correct version 1.0.1 or 1.0.2? The metadata is different from the one here.

Reply:
1.0.2 this has been fixed now